app/src/lib/api.ts

```diff
@@ -47,8 +56,17 @@ interface IFetchAllOptions<T> {
    * `(results) => results.length < 500`
    *
    * @param results  All results retrieved thus far
+   * @param page     The last fetched page of results
    */
```

Comment:
I think this line can be removed
```suggestion
   */
```